tests/src/dir.targets

```diff
@@ -62,17 +62,16 @@
   <Target Name="CoreCompile" />
   
   <!-- If we are a run-only, that depends on another project, this is the "Build" we use. I.e. build all dependency projects, absolutely.
-  OR, in rare conditions (i.e. non-existant ones so far) a run-only script might be something we depend on, and so we may be asked to
-  build a run only for a higher priority testcase.
   -->
-  <Target Name="Build" Condition="('$(CLRTestPriority)' &lt;= '$(CLRTestPriorityToBuild)') Or ('$(_CLRTestInServiceOfHigherPriority)')">
-    <MSBuild Projects="@(ProjectReference)" Properties="_CLRTestInServiceOfHigherPriority=true"/>
+
+  <Target Name="Build" Condition="('$(CLRTestPriority)' &lt;= '$(CLRTestPriorityToBuild)')">
```

Comment:
you should make this condition on 'RunOnly', as of today this is getting silently overridden below